src/main/java/ch/epfl/biop/imaris/EasyXT.java

```diff
@@ -21,18 +21,22 @@
 
 package ch.epfl.biop.imaris;
 
+
 import Ice.ObjectPrx;
 import Imaris.Error;
 import Imaris.*;
 import ImarisServer.IServerPrx;
 import com.bitplane.xt.IceClient;
+
```

Comment:
Extra Space, use "Organise Imports" to clean up this part

Reply:
d75e73671ff45a8fd59996f2127403d2195c3ff4 is this ok ?